lib/extendOrdWithCustom.js

```diff
@@ -47,7 +47,7 @@ function compareAndHandleCustomORDContentWithExistingContent(ordContent, customO
 }
 
 function getCustomORDContent(appConfig) {
-    if (appConfig.env.customOrdContentFile) {
+    if (appConfig.env?.customOrdContentFile) {
```

Comment:
Hi @RamIndia, why do you need this question mark? You can see the [code](https://github.com/cap-js/ord/blob/main/lib/ord.js#L74), there is always env in appConfig.

Reply:
Is this really given? What happens if there's no `ord` dictionary in the `.cdsrc` file? But if you want to fall back to a non-null value, you could do the "default" fallback in the linked file like:

```js
env: cds.env["ord"] || {},
```

Reply:
Good point, I create an issue: https://github.com/cap-js/ord/issues/74